### PR TITLE
Update top forty

### PR DESCRIPTION
**Pull request by ljwhite**
Updates for the PR:

Friends are now listed in the navbar section
Viewing parties are now listed in the navbar section, both user-created and user-invited
Flash messages are now incorporated into the application layout
Top 40 rated now shows 40 movies instead of 20.
Rubocop passes.
Limited merge conflicts will be required

### CODE REVIEW

File: app/views/partials/_nav_bar.html.erb [hunk 1]
```diff
@@ -3,10 +3,36 @@
     <%= button_to "Discover Movies", movies_discover_path, method: :get %>
   </section>
   <section class ="friends">
-    Friends
+    <br><h3>Friends</h3>
```
Comment (twarbelow) [inline app/views/partials/_nav_bar.html.erb:6]: I think this should be a link to a Friends index page

File: app/views/partials/_nav_bar.html.erb [hunk 2]
```diff
@@ -3,10 +3,36 @@
...
+    <% else %>
+      <% current_user.friends.each do |friend| %>
+        <%= friend.first_name%>
+        <% end %>
+    <% end %>
     </section>
+
   <section class ="viewing_parties">
-    Viewing Parties
+    <br><h3>Viewing Parties</h3>
```
Comment (twarbelow) [inline app/views/partials/_nav_bar.html.erb:17]: I think this should be a link to a viewing parties index page.

File: config/routes.rb [hunk 3]
```diff
@@ -13,4 +13,6 @@
   get '/movies/search/results', to: 'movies#index'
   get '/movies/top_rated', to: 'movies#top_rated'
   get "movie_show", to: "movies#show"
+  get "new_viewing_party", to: "view_parties#new"
```
Comment (twarbelow) [inline config/routes.rb:16]: is there a reason the syntax is different for this vs line 17? viewing/view

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,24 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+ Invitee.destroy_all
+ViewParty.destroy_all
```
Comment (twarbelow) [inline db/seeds.rb:10]: indent to match style above and below

File: spec/features/friends/add_and_show_friends_spec.rb [hunk 5]
```diff
@@ -24,6 +24,24 @@
...
+      within("form#friends") do
+        fill_in :email_id, with: "typo@gmail.com"
+        click_button("Add Friend")
+      end
+    expect(current_user.friends.count).to eq(0)
+    expect(friend.friends.include?(current_user)).to be_falsey
+    expect(page).to have_content("There is no user with that email address")
+  end
+
   end
```
Comment (twarbelow) [inline spec/features/friends/add_and_show_friends_spec.rb:47]: is this properly indented?

File: spec/features/movies/top_rated_spec.rb [hunk 6]
```diff
@@ -2,20 +2,11 @@
...
 RSpec.describe "as an authorized user" do
   it "I can go to a discover movies page that has a link to find top movies and a text field to perform a keyword search with a 'Find Movies' button" do
-    VCR.use_cassette('top_rated_content', :match_requests_on => [:method, :path]) do
+    VCR.use_cassette('top_rated_movies', :match_requests_on => [:method, :path]) do
       login_as_user
-
       click_button "Discover Movies"
       click_button("Find Top-Rated Movies")
-      expect(page.all('table tr').count).to eq(21)
+      expect(page.all('table tr').count).to eq(41)
```
Comment (twarbelow) [inline spec/features/movies/top_rated_spec.rb:9]: why is this 41 instead of 40?

File: test/models/invitee_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (twarbelow) [inline test/models/invitee_test.rb:1]: looks like this is an autogenerated file that we aren't using. safe to delete?

File: test/models/view_party_test.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (twarbelow) [inline test/models/view_party_test.rb:1]: looks like this is an autogenerated file that we aren't using. safe to delete?

File: app/views/partials/_nav_bar.html.erb (hunk 1 above)
Comment (ljwhite) [reply app/views/partials/_nav_bar.html.erb]: Moved to the welcome page

File: app/views/partials/_nav_bar.html.erb (hunk 2 above)
Comment (ljwhite) [reply app/views/partials/_nav_bar.html.erb]: Moved to the welcome page

File: config/routes.rb (hunk 3 above)
Comment (ljwhite) [reply config/routes.rb:16]: Specific path to get to the form.

File: spec/features/friends/add_and_show_friends_spec.rb (hunk 5 above)
Comment (ljwhite) [reply spec/features/friends/add_and_show_friends_spec.rb:47]: Fixed

File: spec/features/movies/top_rated_spec.rb (hunk 6 above)
Comment (ljwhite) [reply spec/features/movies/top_rated_spec.rb:9]: Headers count as a row

File: test/models/invitee_test.rb (hunk 7 above)
Comment (ljwhite) [reply test/models/invitee_test.rb:1]: Correct, autogenerated. I don't know if it's ok to delete, or if it should stay.

File: test/models/view_party_test.rb (hunk 8 above)
Comment (ljwhite) [reply test/models/view_party_test.rb:1]: Correct, autogenerated. I don't know if it's ok to delete, or if it should stay.

File: db/seeds.rb (hunk 4 above)
Comment (ljwhite) [reply db/seeds.rb:10]: Fixed

File: test/models/invitee_test.rb (hunk 7 above)
Comment (twarbelow) [reply test/models/invitee_test.rb:1]: I think we could probably delete, but I'll try it on a branch later.

**Pull request merged**